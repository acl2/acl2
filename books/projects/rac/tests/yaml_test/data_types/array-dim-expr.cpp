// RAC begin

int foo() {
  int a[1 + 1] = {0, 0, 0};
  return 0;
}
