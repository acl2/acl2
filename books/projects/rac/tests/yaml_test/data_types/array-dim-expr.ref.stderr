array-dim-expr.cpp:4 (6-7): Too many arguments, expected 2 argument(s) got 3
 4 |   int a[1 + 1] = {0, 0, 0};
   |       ^
